### ISSUE TITLE
ctrl-shift-del not working to delete test in list

### ISSUE THREAD

**Issue by coinzdude** [SeleniumHQ/selenium-ide#1618]
the menu works here but not the key chord.

![image](https://user-images.githubusercontent.com/58857646/231818871-d4736961-ef75-4df0-a6f0-174f6275557b.png)

**Comment by toddtarsi** [comment]
@coinzdude - Oh, I see! Does Ctrl + Shift + Del do something in the other list? Maybe this list is never active and so the hotkeys are getting consumed in the main list instead.

**Comment by coinzdude** [comment]
Good catch, it has the same issue in suites management too. I agree it is likely getting consumed in the main list as both have a focus/highlight indicator at the time. 

Not sure what to do with it yet, I might see if the focused window can bubble up the event for certain keys.
